src/components/PullRequestList/PullRequestListItem/index.js

```diff
@@ -1,11 +1,11 @@
-/* @flow */
+// TODO: add flow annotations
```

Comment:
I guess you know what I want to say here... Let me know if there is a specific reason why you disabled it for now.